nilearn/decomposition/tests/test_multi_pca.py

```diff
@@ -55,3 +55,46 @@ def test_multi_pca():
 
     # Smoke test to fit with no img
     assert_raises(TypeError, multi_pca.fit)
+
+def test_multi_pca_score():
+    shape = (6, 8, 10, 5)
+    affine = np.eye(4)
+    rng = np.random.RandomState(0)
+
+    # Create a "multi-subject" dataset
+    data = []
+    for i in range(8):
+        this_data = rng.normal(size=shape)
+        # Create fake activation to get non empty mask
+        this_data[2:4, 2:4, 2:4, :] += 10
```

Comment:
Why do you have a constant patch (instead of plain noise)? I think that this is not used in the test. Sorry if I missed something obvious.